CharacterCreation/02_Classes.txt

```diff
@@ -3,14 +3,12 @@ Core classes:
  * Your character starts at level 0. At level 0 you have skills, but no class, 
  * class feats or regular feats.
  * At level 1 you get to select a class, and receive the class passives. 
- * However, you do not select a class feat until level 5.
- * On selecting a class, you gain all of the passives listed, and get to 
+ * However, you do not select a class feat until level 5. At level 5 and every 5
+ * levels thereafter, you can gain a class feat.
+ *
  * For any listed class, if it says "Gain X Skill at 10" that means you get
- * to add the X skill to your skill list, and it starts at rank 10, meaning a 
- * +10% to any roll for that skill. At the first level taken in a class
- * (other than level 1), you get the single star version of the feat. At 
- * level 5 and every 5 levels thereafter, you can choose to rank up the 
- * feat, or add the other feat to your  character.
+ * to add the X skill, at 10 points, to your skill list. Thereby gaining a +10% to
+ * any roll which uses that skill. 
```

Comment:
Can we reword this snippet? While I don't care all that much about comas, a sentence should not be made out of four sentence fragments.

Reply:
#199. Well, it looks like you invalidated your own comment by making a separate pull request to change something i'm already changing?